content/backstage/plugins/jenkins.md

```diff
@@ -33,8 +33,6 @@ coverImageAlt: 'A screenshot of the Jenkins plugin.'
 availableOnRoadie: true
 roadieDocsPath: /integrations/jenkins/
 
-thingsToKnowTitle: How to get a Jenkins API token
-
```

Comment:
This property was used to selectively update one of the headings in the plugin directory pages. It was only ever used on 3 out of ~50 plugins, and was completely unnecessary. I'm removing the code path.